lang/en/admin/databasehost.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+return [
+    'create_action' => ':action Database Host',
```

Comment:
Resources should not be translated, probably can use a single `lang/en/admin/reource.php` key for these.

```suggestion
    'create_action' => ':action :resource',
```